python/air/compiler/aircc/main.py

```diff
@@ -358,6 +358,7 @@ def run_flow(opts):
       cmd += [f'-I{thispath}/../../../../runtime_lib/airhost/include']
       cmd += [f'-I{thispath}/../../../../runtime_lib']
       cmd += [f'-I{thispath}/../../../../../aie/runtime_lib']
+      cmd += [f'-I{thispath}/../../../../../utils/mlir-aie/runtime_lib']
```

Comment:
This just adds a second hard coded path to mlir-aie? I'd rather fix this properly (see https://github.com/Xilinx/mlir-air/issues/30)

Reply:
If mlir-aie and mlir-air are installed to the same place then line 359 should be sufficient. i.e. they will share a runtime_lib install directory.

Reply:
I don't believe the building documentation states to install them in the same directory. I was thinking of just adding some logic similar to the Makefile (https://github.com/Xilinx/mlir-air/blob/main/test/13_mb_add_one/Makefile#L5) to find where mlir-aie was installed. Thoughts?

Reply:
A few thoughts:
 - Put this in for now but maybe we should update the documentation to not suggest such a strange install location for mlir-aie? This assumes that aircc is installed relative to the mlir-air source directory which in turn has mlir-aie installed inside it.
 - The relative path might be different for the lit tests because testing runs out of the build directory not an install (because test comes before install). I think line 360 even assumes a certain test setup.
 - Finding the path with `which aie-opt` is an option and is effectively how we find `aie-opt, aie-translate,` and `aiecc.py` But cmake also knows the location of the mlir-aie it's building against and can set the path at compile time.